drivers/tee/optee/optee_smc.h

```diff
@@ -383,19 +383,19 @@ struct optee_smc_disable_shm_cache_result {
 	OPTEE_SMC_RPC_VAL(OPTEE_SMC_RPC_FUNC_FREE)
 
 /*
- * Deliver an IRQ in normal world.
+ * Deliver a foreign interrupt in normal world.
```

Comment:
Sorry, paste error. It should have been *"Deliver foreign interrupt to normal world"*.
Once we're in normal world it's not a foreign interrupt anymore.
I'm only suggesting to think about it, if you agree please change otherwise keep it as it is.